src/backend/EdFi.DataManagementService.Backend.Postgresql/Deploy/Scripts/1000_Create_Reference_Validation_FKs.sql

```diff
@@ -0,0 +1,10 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+-- Reference validation enforcement occurs via this constraint
+-- Omitting this FK effectively disables reference validation
+ALTER TABLE "references"
+ADD CONSTRAINT FK_References_ReferencedAlias FOREIGN KEY (ReferentialPartitionKey, ReferentialId)
+REFERENCES Aliases (ReferentialPartitionKey, ReferentialId) ON DELETE RESTRICT;
```

Comment:
This is the reference validation FK. Declared as ON DELETE RESTRICT more as documentation for emphasis than anything else.